### PR TITLE
real_time_config_quick_scan:unstable-2020-08-03 -> unstable-2020-07-23

### PR DESCRIPTION
**Pull request by magnetophon**
also resolve suggestions from https://github.com/NixOS/nixpkgs/pull/78998

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/real_time_config_quick_scan/default.nix [hunk 1]
```diff
@@ -24,20 +25,25 @@ stdenv.mkDerivation rec {
     install -Dm 755 "$i" "$out/share/$i"
     done
     # Install doc files:
-    install -D COPYING  "$out/share/doc/COPYING"
+    install -D COPYING  "$out/share/doc/$pname/COPYING"
     install -D README.md  "$out/share/doc/README.md"
```
Comment (jtojnar) [inline pkgs/applications/audio/real_time_config_quick_scan/default.nix:29]: Same here.

File: pkgs/applications/audio/real_time_config_quick_scan/default.nix [hunk 2]
```diff
@@ -24,20 +25,25 @@ stdenv.mkDerivation rec {
     install -Dm 755 "$i" "$out/share/$i"
```
Comment (jtojnar) [inline pkgs/applications/audio/real_time_config_quick_scan/default.nix:25]: And here.

Comment (magnetophon) [reply pkgs/applications/audio/real_time_config_quick_scan/default.nix]: Oops, sorry!

**Comment by jtojnar** [comment]
Why the downgrade?

**Comment by jtojnar** [comment]
Thanks.

**Pull request merged**